### PR TITLE
Local prod

### PR DESCRIPTION
**Pull request by wan9c9**
fixed a typo and added a logger.info to log more info.

## How Has This Been Tested?

- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [x] Fix bugs
- [ ] Add new feature
- [ ] Update documentation

### CODE REVIEW

File: qlib/data/ops.py [hunk 1]
```diff
@@ -1569,7 +1569,6 @@ def _load_internal(self, instrument, start_index, end_index, *args):
 
 TOpsList = [TResample]
 OpsList = [
-    ChangeInstrument,
```
Comment (you-n-g) [inline qlib/data/ops.py:1572]: Hi, Why do you remove this operator?

**Review by you-n-g — COMMENTED** [review]
Please fix the CI Error.
Thanks

File: qlib/data/ops.py (hunk 1 above)
Comment (wan9c9) [reply qlib/data/ops.py:1572]: Oh, I didn't mean to remove it. I'll fix it. Thanks!

**Comment by wan9c9** [comment]
Closing the pull request as there is some error, and made a new pull request.

**Pull request closed without merging**